function-logs/handler.go

```diff
@@ -0,0 +1,99 @@
+package function
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/openfaas/openfaas-cloud/sdk"
+)
+
+// Handle grabs the logs for the fns that are named in the input
+func Handle(req []byte) string {
```

Comment:
If I get the URL for this function somehow, can I just get logs from whatever I want providing I craft the correct querystring?

Reply:
At the least, perhaps we need to invoke list-functions passing in the user to see if the function is:

A) returned
B) having the git-owner label? (implied by A)

Otherwise, if I got internal access I could invoke this and get logs from any of the system/pipeline functions (I think).

Reply:
this has been addressed by calling list-function and checking that the fn is in there, and they are the git-owner (by label)